src/ext.rs

```diff
@@ -48,15 +48,16 @@ pub fn externalize_mem(mut module: elements::Module, adjust_pages: Option<u32>,
 		entry = elements::MemoryType::new(entry.limits().initial(), Some(max_pages));
 	}
 
-	import_section(&mut module).expect("Import section to exist").entries_mut().push(
+	let mut builder = builder::from_module(module);
+	builder.push_import(
```

Comment:
so it's ok if ImportSection exists already?

Reply:
yep